ubuntu-advantage

```diff
@@ -75,7 +75,9 @@ Commands:
  enable-livepatch <TOKEN>  enable the Livepatch service
  disable-livepatch [-r]    disable the Livepatch service. With "-r", the
                            canonical-livepatch snap will also be removed
-
+ update-fips <token> [-y]  Install and enable updated non-certified FIPS
+                           modules. With "-y" or "--yes" the user prompt
+                           will be bypassed.
```

Comment:
Please keep this close to other fips command entries (and in the same order as the manpage)

Reply:
Fixed.